new lint/format feature (#3459)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-versions/release-notes/07-June-2023/lint-format-rn.md

---
title: "New: You can now lint and format your code in the IDE"
id: "lint-format-rn"
description: "June 2023 release note: In the dbt Cloud IDE, you can perform linting and formatting on SQL, YAML, Markdown, Python, and JSON files using tools like SQLFluff, sqlfmt, Prettier, and Black."
sidebar_label: "New: Lint and format in the IDE"
sidebar_position: 1
tags: [June-2023, IDE]
---

dbt Labs is excited to announce you can now lint and format your dbt code in the dbt Cloud IDE. This is an enhanced development workflow which empowers you to effortlessly prioritize code quality. 

> In the dbt Cloud IDE, you now have the capability to perform linting and formatting on five different file types: SQL, YAML, Markdown, Python, and JSON. 

Reviewer: This is repeating the opening statement. Maybe we can combine the two. 
```suggestion
You can perform linting and formatting on five different file types: SQL, YAML, Markdown, Python, and JSON. 
```

Author: i just love saying the ide all the time 😅
